jobs/taar_similarity.py

```diff
@@ -30,103 +33,32 @@
 from pyspark.mllib.stat import KernelDensity
 from pyspark.statcounter import StatCounter
 from scipy.spatial import distance
-import boto3
 from google.cloud import bigquery
 
 
-def aws_env_credentials():
-    """
-    Load the AWS credentials from the enviroment
-    """
-    result = {
-        "aws_access_key_id": os.environ.get("AWS_ACCESS_KEY_ID", None),
-        "aws_secret_access_key": os.environ.get("AWS_SECRET_ACCESS_KEY", None),
-    }
-    return result
-
-
-@contextlib.contextmanager
-def selfdestructing_path(dirname):
-    import shutil
-
-    yield dirname
-    shutil.rmtree(dirname)
-
+def read_from_gcs(fname, prefix, bucket):
```

Comment:
at quick glance, it seems the reading/writing to gcs code is duplicated and could be written once as a common lib, but upto you again

Reply:
yes, I noticed this. this PR is based on the original Victor's work, and I didn't change much, just fixed bugs. It would be a good idea of course but requires some development time which we don't have. I can add it as todo for future